src/content/docs/tutorial-kubernetes-learn/tutorial-k8s-layers.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: Simplify kubernetes layers with New Relic
+---
+
+import k8sDiagramContext from 'images/tutorials_diagram_kubernetes-overview.webp'
+
+import tutorialOverviewdashboard from 'images/tutorial_screenshot-full_overview-dashboard.webp'
+
+import tutorialOverviewDeployments from 'images/tutorial_screenshot-full_overview-deployments.webp'
+
+import tutorialOverviewFailed from 'images/tutorial_screenshot-crop_failed-pods.webp'
+
+import tutorialAPMOverview from 'images/tutorial_screenshot-full_apmK8s.webp'
+
+import tutorialAPMPerformance from 'images/tutorial_screenshot-full_apm-k8s-performance.webp'
+
+Understanding the inner workings of your Kubernetes system, specifically how all the different functions of the system come together, is inherently complex. Let's review what makes up a Kubernetes system and then dive into how New Relic can help monitor and understand each part.
+
+## Break it apart to understand
+
+To gain a better grasp of a Kubernetes system, let's break it down into distinct layers and explore how New Relic can help you comprehend each one effectively.
+
+<img
+    title="k8sDiagramContext"
+    alt="an image showing an abstracted view of a kubernetes system. This includes pods, apps, and clusters"
+    src={k8sDiagramContext}
+/>
+
+We'll discuss a Kubernetes system in three key sections:
```

Comment:
It seems super important that you get this next part 100% correct. You could lose a lot of credibility here if it's not rock solid.

Reply:
Having read these three sections now, I think they need a bit more polishing/refining.

If I'm a reader experienced with Kubernetes, I want you to show me you know what you're writing about.

If I'm a Kubernetes tyro, I want this to be super clear so that you've demystified it for me.